google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudType.java

```diff
@@ -91,7 +92,7 @@ public AppEngineServerConfiguration createDefaultConfiguration() {
   @Override
   public RemoteServerConfigurable createServerConfigurable(
       @NotNull AppEngineServerConfiguration configuration) {
-    return new AppEngineCloudConfigurable(configuration, null);
```

Comment:
I'm just wondering if the server configuration model is something we want to bypass or whether we should proxy the cloud sdk service through the configuration model.